### PR TITLE
Handle change tracking of log items

### PR DESCRIPTION
**Pull request by lbodor**
Persist site log items along with their change tracking attributes, but disregard deleted log items when computing setups.

Note that this pull request is stacked on top of #505.

### CODE REVIEW

File: gws-core/src/main/java/au/gov/ga/geodesy/domain/model/sitelog/EquipmentLogItem.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package au.gov.ga.geodesy.domain.model.sitelog;
 
-public interface EquipmentLogItem extends LogItem {
-    String getType();
-    String getSerialNumber();
+public abstract class EquipmentLogItem extends LogItem {
```
Comment (oehm-smith) [inline gws-core/src/main/java/au/gov/ga/geodesy/domain/model/sitelog/EquipmentLogItem.java]: Why change to an abstract class - you aren't adding any base methods?

Comment (lbodor) [inline gws-core/src/main/java/au/gov/ga/geodesy/domain/model/sitelog/EquipmentLogItem.java]: LogItem is no longer an interface.

File: gws-core/src/main/java/au/gov/ga/geodesy/domain/service/CorsSiteService.java [hunk 2]
```diff
@@ -132,7 +133,12 @@ public void handle(SiteLogReceived siteLogReceived) {
         Comparator<Instant> fromC = ComparatorUtils.nullLowComparator(ComparatorUtils.NATURAL_COMPARATOR);
         SortedSet<Instant> datesOfChange = new TreeSet<>(fromC);
 
-        for (EquipmentLogItem logItem : siteLog.getEquipmentLogItems()) {
+        List<EquipmentLogItem> equipmentLogItems = siteLog.getEquipmentLogItems()
```
Comment (oehm-smith) [inline gws-core/src/main/java/au/gov/ga/geodesy/domain/service/CorsSiteService.java]: Nice.  I like this way.

**Comment by oehm-smith** [comment]
:+1:

**Pull request merged**